src/scmdata/netcdf.py

```diff
@@ -290,6 +292,10 @@ def run_to_nc(df, fname, dimensions=("region",), extras=()):
     with nc.Dataset(fname, "w", diskless=True, persist=True) as ds:
         ds.created_at = datetime.utcnow().isoformat()
         ds._scmdata_version = __version__
+
+        if hasattr(df, "metadata"):
```

Comment:
Naming runs as df is a little bit weird, refactoring can wait though if you don't want to flick it across now.